src/OpenSage.DataViewer/ViewModels/ShellViewModel.cs

```diff
@@ -38,18 +37,11 @@ public ShellViewModel()
 
         private static IEnumerable<InstallationViewModel> FindInstallations()
         {
-            var programFilesPath = Environment.GetFolderPath(Environment.SpecialFolder.ProgramFilesX86);
+            var locator = IoC.Get<IInstallationLocator>();
 
-            var possibleInstallations = new[]
-            {
-                new InstallationViewModel("C&C Generals (The First Decade)", Path.Combine(programFilesPath, @"EA Games\Command & Conquer The First Decade\Command & Conquer(tm) Generals")),
-                new InstallationViewModel("C&C Generals (Origin)", Path.Combine(programFilesPath, @"Origin Games\Command and Conquer Generals Zero Hour\Command and Conquer Generals")),
-                new InstallationViewModel("C&C Generals Zero Hour (The First Decade)", Path.Combine(programFilesPath, @"EA Games\Command & Conquer The First Decade\Command & Conquer(tm) Generals Zero Hour")),
-                new InstallationViewModel("C&C Generals Zero Hour (Origin)", Path.Combine(programFilesPath, @"Origin Games\Command and Conquer Generals Zero Hour\Command and Conquer Generals Zero Hour")),
-                new InstallationViewModel("Battle for Middle-earth (DVD)", Path.Combine(programFilesPath, @"EA Games\The Battle for Middle-earth (tm)"))
-            };
-
-            return possibleInstallations.Where(x => Directory.Exists(x.Path));
+            return Games.GetAll()
+                .SelectMany(locator.FindInstallations)
+                .Select(installation => new InstallationViewModel(installation.DisplayName, installation.Path));
```

Comment:
There could be multiple different installations of the same game, and there is currently no way to distinguish between them in the UI.